src/sbml/ListOf.h

```diff
@@ -604,6 +604,11 @@ class LIBSBML_EXTERN ListOf : public SBase
 
 
   /** @endcond */
+
+  /** @cond doxygenLibsbmlInternal */
+  void sort();
```

Comment:
ideally there would be a docstring here, detailing that this function sorts the list based on the id of the Sbase elements.